pages.zh/common/gpg.md

```diff
@@ -0,0 +1,54 @@
+# gpg
+
+> GNU Privacy Guard.
+> 请参阅 `gpg2` 了解 GNU Privacy Guard 2。
+> 更多信息：<https://gnupg.org>。
+
+- 不加密，仅对 doc.txt 进行签名(生成 doc.txt.gpg，格式为二进制):
+
+`gpg --sign {{doc.txt}}`
+
+- 不加密，仅对 doc.txt 进行签名，生成单独的签名文件 (生成 doc.txt.sig，格式为二进制):
+
+`gpg --detach-sign {{doc.txt}}`
+
+- 不加密，仅对 doc.txt 进行签名(生成 doc.txt.asc，格式为 ASCII 码形式):
+
+`gpg --clearsign {{doc.txt}}`
+
+- 不加密，仅对 doc.txt 进行签名,生成单独的签名文件(生成 doc.txt.asc，格式为 ASCII 码形式):
+
+`gpg --armor --detach-sign {{doc.txt}}`
+
+- 加密+签名
+
+`gpg --local-user [发信者ID] --recipient [接收者ID] --armor --sign --encrypt {{doc.txt}}`
+
+- 验证签名：
```

Comment:
There's that strange colon again

Reply:
Google translates this to "Verify signature" which again sounds a little too terse&mdash;how about the Chinese equivalent of this?
```
- Verify the signature of `doc.txt`:
```